app/Api/Controllers/DashboardWidgetController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Api\Controllers;
+
+use App\Models\Widgets;
+use App\Models\UsersWidgets;
+use Dingo\Api\Http;
+use Dingo\Api\Routing\Helpers;
+use Illuminate\Http\Request;
+
+class DashboardWidgetController extends Controller
+{
+
+    use Helpers;
+
+    public function __construct() {
+
+    }
+
+    /**
+     * Display a listing of all authorized devices
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $row                         = UsersWidgets::where('user_id', $request->user()->user_id)->where('dashboard_id', $request->dashboard_id)->max('row')+1;
+        $user_widget                 = new UsersWidgets;
+        $user_widget->user_id        = $request->user()->user_id;
+        $user_widget->widget_id      = $request->widget_id;
+        $user_widget->col            = $request->col;
+        $user_widget->row            = $row;
+        $user_widget->size_x         = $request->size_x;
+        $user_widget->size_y         = $request->size_y;
+        $user_widget->title          = $request->title;
+        $user_widget->dashboard_id   = $request->dashboard_id;
+        if ($user_widget->save())
+        {
+            return $this->response->array(array('statusText' => 'OK', 'user_widget_id' => $user_widget->user_widget_id));
+        }
+        else {
+            return $this->response->errorInternal();
+        }
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Request $request, $id)
+    {
+        $widget  = Widgets::find($id);
+        $content = $this->api->be(auth()->user())->get('/api/dashboard-widget/' . $id . '/content');
+        return array('widget' => $widget, 'content' => $content);
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        //
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, $id)
+    {
+        $users_widgets         = UsersWidgets::find($id);
```

Comment:
No, but you could make the input names match up right?  Only fillable fields will be allowed (to avoid injection).

I only suggested it to make sure you were aware it is possible.  Either way is fine by me.